_data/snippets.yml

```diff
@@ -34,3 +34,30 @@ trusty_note_no_osx: |
   > information are in our reference pages:
   >  * [Precise](/user/reference/precise/)
   >  * [Trusty](/user/reference/trusty/)
+
+contact_enterprise_support: |
+  ## Contact Enterprise Support
+  
+  To get in touch with us, please write a message to 
+  [enterprise@travis-ci.com](mailto:enterprise@travis-ci.com). If possible, 
+  please include as much of the following info as you:
+   
+  - What is the problem?
+  - Which steps did you try already?
+  - A support bundle (You can get it from `https://yourdomain:8800/support`)
+  - Worker log files (They can be found at `/var/log/upstart/travis-worker.log`) 
+  - If you're using multiple worker machines, we need the log files from all of them.
+  
+  Is anything special with your setup? While we may be able to see some 
```

Comment:
Should we reword "Is anything special with your setup?" to something like "Have you made any particular customizations to your setup?" or something similar?

Maybe use long form of Infrastructure as a Service?

Reply:
I like this! used `Have you made any customizations to your setup?`